gplugins/lumerical/recipes/fdtd_recipe.py

```diff
@@ -0,0 +1,204 @@
+import hashlib
+from pathlib import Path
+
+import gdsfactory as gf
+import pandas as pd
+from gdsfactory import Component
+from gdsfactory.config import logger
+from gdsfactory.pdk import LayerStack, get_layer_stack
+from gdsfactory.typings import PathType
+
+from gplugins.design_recipe.DesignRecipe import DesignRecipe, eval_decorator
+from gplugins.lumerical.convergence_settings import (
+    LUMERICAL_FDTD_CONVERGENCE_SETTINGS,
+    ConvergenceSettingsLumericalFdtd,
+)
+from gplugins.lumerical.fdtd import LumericalFdtdSimulation
+from gplugins.lumerical.simulation_settings import (
+    SIMULATION_SETTINGS_LUMERICAL_FDTD,
+    SimulationSettingsLumericalFdtd,
+)
+
+
+def example_run_fdtd_recipe():
```

Comment:
maybe better to have this in an examples file/dir given it's like half of the total file length

Reply:
Done:
82f6483c7ab588d534fba4a96118158e28dbd3e3